base/ltdirchk.dtx

```diff
@@ -36,8 +36,10 @@
 %</unstripped>
 %<*driver>
 % \fi
+% \fi
 \ProvidesFile{ltdirchk.dtx}
-             [2021/12/11 v1.3a LaTeX Kernel (System Dependent Parts)]
+             [2024/02/11 v1.3a LaTeX Kernel (System Dependent Parts)]
+% \iffalse
```

Comment:
how comes we need two \fi and two \iffalse? (might be correct ...can't check now but looks odd)

Reply:
as above I left it that way  as that's what we did for all the original ones `ltclass.dtx`, `ltdefns.dtx`  etc all have  separate but adjacent `\iffalse` around the preamble text then the driver block. I think historically it allowed some experiments with moving the way the docstrip guards worked.